src/js/components/RadioButton/doc.js

```diff
@@ -65,9 +65,14 @@ export const themeDoc = {
   },
   'radioButton.border.width': {
     description: 'The width size of the border of the RadioButton.',
-    type: 'string',
+    type: 'string | {dark: string, light: string}',
     defaultValue: '2px',
   },
+  'radioButton.check.background.color': {
+    description: 'The background color of the checked icon in the RadioButton.',
+    type: 'string',
```

Comment:
Per comment above, I think this color object was applied in the wrong location.

```suggestion
    type: 'string | {dark: string, light: string}',
```

Reply:
Good catch.

Reply:
Good catch @halocline, thanks.